tests/database_operations_test.py

```diff
@@ -0,0 +1,79 @@
+import os
+import typing
+
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from skate_tricks.configs.db_config import get_db_settings, create_db
+from skate_tricks.configs.db_setup import wipe_db_data
+from skate_tricks.database_operations import get_all_tricks, post_new_trick
+from skate_tricks.schemas import postgres_models as models
+from skate_tricks.schemas import pydantic_schemas as json_schemas
+
+os.environ["DATABASE_HOST"] = "127.0.0.1"
+os.environ["DATABASE_PORT"] = "5432"
+os.environ["DATABASE_NAME"] = "tricks_db_mock"
+
+if __name__ == "__main__":
+    create_db()
+
+# TODO: problem with alembic and dropping/recreating enums through upgrades and downgrades...
+
+
+def pass_test_session(func):  # type: ignore # TODO: work out type
```

Comment:
This is so close to being really great :) 

It's a nice idea to create this dependency convenience so that you reduce the amount of code.

I think pytest has a specific way to allow you to 'inject dependencies' in to your tests. You should have a look at that. There are a couple of roufgh edges to your implementation which would be helped by using that framework. Can discuss more if you don't find what I'm suggesting.